### ISSUE TITLE
Friendly names for known contracts

### ISSUE THREAD

**Issue by menduz** [decentraland/proposals#151]
As we can address contracts in the world's entities, we will need a friendly way to identify and address known contracts.

The proposed way is to deploy a smart contract that maps a lower case friendly name to a smart contract address.
* This mapping must be updateable
* This mapping must be revocable

(@eordano @belohlavek please complete with more detailed requirements and specifications)

**Comment by belohlavek** [comment]
Use case: As a user I want to reference my ERC721/821 collectible in my XML parcel definition.

- If we hardcode contract addresses, then updating all parcels if the address changes is going to be a mess. This is a plausible scenario because we are deploying proxy contracts for non ERC821 CCs, we may need to redeploy new ones with updates in the future.

- We want to keep the client generic enough and avoid having specific strategies for certain collectibles.

**Comment by chentschel** [comment]
I added a new repo for this at https://github.com/decentraland/contracts-locator. 
- It supports adding a contract address under a a namespace (string), and optionally a ttl value.  
- We'll be uploading proxies for registries not supporting / ERC821-721 to the proxies folder. 
- @eordano should we set some guides on namespace conventions?. 
- Also, should we add **'admin'** addresses capable of setting / removing services to the contract. (currently only the contract owner)
